server/src/test/java/org/elasticsearch/index/query/QueryStringQueryBuilderTests.java

```diff
@@ -84,6 +85,7 @@
 import static org.hamcrest.Matchers.instanceOf;
 
 public class QueryStringQueryBuilderTests extends FullTextQueryTestCase<QueryStringQueryBuilder> {
```

Comment:
Can we remove the `FullTextQueryTestCase` entirely if we ensure that `now` cannot be part of the query string.
You'll need to add dedicated tests in MatchQueryBuilderTests and MultiMatchQueryBuilderTests but we need to do that anyway so... There is a check in `FullTextQueryTestCase` that is related to fuzziness but it is wrong and should be removed anyway (fuzziness doesn't disable caching).